File: PspDataPresenter.Core/Models/AgendaModel.cs
using System.Collections.Generic;

namespace PspDataPresenter.Core
{
    public class AgendaModel
    {
        public int Id { get; set; }
        public int ScheduleNo { get; set; }
        public string Name { get; set; }
        //standard, interpellation
        public string Type { get; set; }
        // active, cancelled
        public string State { get; set; }
        public string Abbreviation { get; set; }
        public string Description { get; set; }
        public string Print { get; set; }
        public List<DebateModel> Debates { get; set; }
        public List<InterpellationModel> Interpellations { get; set; }
    }
}

File: PspDataPresenter.App/data-presenter-app/src/app/_core/models/agenda.model.ts
import { DebateModel } from "./debate.model";
import { InterpellationModel } from "./interpellation.model";

export interface AgendaModel {
    id: number;
    scheduleNo: number;
    name: string;
    type: string;
    state: string;
    abbreviation: string;
    description: string;
    print: string;
    debates: DebateModel[];
    interpellations: InterpellationModel[];
}